sound.js

```diff
@@ -117,10 +117,36 @@ Sound.prototype.stop = function(callback) {
   return this;
 };
 
-Sound.prototype.reset = function() {
+Sound.prototype.setStreamType = function(streamType, callback) {
+  // For android only.
+  // Resets current mediaPlayer, change streamType, and prepare again.
```

Comment:
How about including `reset` in there?

Reply:
@ghsdh3409 Do you mean `setStreamType` function should include both resetting functionality and setting stream type functionality? Sorry for the wrong comment in line 122. This function currently does not `resets` the mediaPlayer.

Reply:
As I remember, we check that Android can't change stream type if a player does not be reset. If so, we have to call `reset` before using `setStreamType`. My proposal is that how about that `setStreamType` do `RNSound.reset` before `RNSound.prepare`. If so, we just use `setStreamType` without calling `reset`.

Reply:
Thanks! I'm totally convinced.